gatsby-config.js

```diff
@@ -54,3 +54,10 @@ module.exports = {
     },
   ],
 }
+module.exports = {

```

Comment:
You don't need to add another `module.exports`. You can simply the contents you would like to add to the `module.exports` already present.

Reply:
Oh I see! Let me rectify it.